src/components/vuestic-components/va-option-list/VaOptionList.demo.vue

```diff
@@ -0,0 +1,167 @@
+<template>
+  <VbDemo>
+    <VbCard
+      title="Options as array of string"
+    >
+      <va-option-list
+        :options="baseData.options"
+        :default-value="baseData.options[0]"
+        color="primary"
+        type="radio"
+        v-model="baseData.selected"
+      />
+      Selected {{ baseData.selected }}
+    </VbCard>
+    <VbCard
+      title="Disable all"
+    >
+      <va-option-list
```

Comment:
Please add v-model for here. That will allow us to test disabled works as intended and value is not emitted.